pkg/local_object_storage/metabase/put.go

```diff
@@ -142,6 +143,14 @@ func (db *DB) put(
 		}
 	}
 
+	if !isParent {
+		err = db.updateCounter(tx, 1, true)
+		if err != nil {
+			db.log.Error("could not increase object counter: %w",
+				zap.Error(err))
```

Comment:
Here too. I think we can (should) return the error to the caller, `updateCounter` has no logical errors.